Content.Server/Light/Components/ToggleableOccluderComponent.cs

```diff
@@ -0,0 +1,21 @@
+using Content.Shared.DeviceLinking;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Light.Components
+{
+    /// <summary>
+    ///     Allows entities with OccluderComponent to toggle that component on and off.
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class ToggleableOccluderComponent : Component
+    {
+        [DataField("onPort", customTypeSerializer: typeof(PrototypeIdSerializer<SinkPortPrototype>))]
+        public string OnPort = "On";
```

Comment:
```suggestion
        [DataField]
        public ProtoId<SinkPortPrototype> OnPort = "On";
```
same with the others